### ISSUE TITLE
Any way to handle external functions from ink story?

### ISSUE THREAD

**Issue by SwastikBhattacharyya** [The-Chinese-Room/Inkpot#30]
I checked the `story.h` file from the inkpot directory and there seems to be no support for binding external functions. Will there be any future support for binding external functions directly from ink?

**Comment by TCR-Nick** [comment]
@SwastikBhattacharyya 
No, this is one part of the base ink functionality that we did not implement as we simply did not need it in our usage case.
see https://github.com/The-Chinese-Room/Inkpot#what-we-did-not-do

We may look to put this in at some point, but unable to give you a time frame on that at the moment.

**Comment by TCR-Nick** [comment]
@SwastikBhattacharyya external functions are now implemented in the lates version of Inkpot.
